operator/v1/types_csi_cluster_driver.go

```diff
@@ -168,6 +168,63 @@ type AWSCSIDriverConfigSpec struct {
 	// +kubebuilder:validation:Pattern:=`^arn:(aws|aws-cn|aws-us-gov|aws-iso|aws-iso-b|aws-iso-e|aws-iso-f):kms:[a-z0-9-]+:[0-9]{12}:(key|alias)\/.*$`
 	// +optional
 	KMSKeyARN string `json:"kmsKeyARN,omitempty"`
+
+	// efsVolumeMetrics sets the configuration for collecting metrics from EFS volumes used by the EFS CSI Driver.
+	// +openshift:enable:FeatureGate=AWSEFSDriverVolumeMetrics
+	EFSVolumeMetrics AWSEFSVolumeMetrics `json:"efsVolumeMetrics"`
```

Comment:
This is optional right? So will need a `+optional` tag, and omitempty.

Since this has a required field within it, it must also be a pointer, else anything that marshals this from JSON will struggle